client/hotrod-client/src/test/java/org/infinispan/client/hotrod/query/aggregation/RemoteClusteredAggregationExpiringTest.java

```diff
@@ -0,0 +1,129 @@
+package org.infinispan.client.hotrod.query.aggregation;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.infinispan.configuration.cache.IndexStorage.FILESYSTEM;
+import static org.infinispan.query.aggregation.QueryAggregationExpiringTest.resultMaps;
+import static org.infinispan.server.hotrod.test.HotRodTestingUtil.hotRodCacheConfiguration;
+
+import java.io.File;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import org.infinispan.client.hotrod.RemoteCache;
+import org.infinispan.client.hotrod.test.MultiHotRodServersTest;
+import org.infinispan.commons.api.query.Query;
+import org.infinispan.commons.dataconversion.MediaType;
+import org.infinispan.commons.test.CommonsTestingUtil;
+import org.infinispan.commons.test.annotation.TestForIssue;
+import org.infinispan.commons.util.Util;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.cache.StorageType;
+import org.infinispan.persistence.dummy.DummyInMemoryStoreConfigurationBuilder;
+import org.infinispan.protostream.SerializationContextInitializer;
+import org.infinispan.query.model.Task;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+@Test(groups = "functional", testName = "client.hotrod.query.aggregation.RemoteClusteredAggregationExpiringTest")
+@TestForIssue(githubKey = "13194")
+public class RemoteClusteredAggregationExpiringTest extends MultiHotRodServersTest {
```

Comment:
Can it be reproduced in a non-clustered environment?

Reply:
If I remember correcty, it is not possible. But I can check

Reply:
I checked the issue is not reproducible without the clustering configuration.